src/components/workspace-utils.js

```diff
@@ -123,6 +123,7 @@ export const SnapshotInfo = ({
   // Lifecycle
   useOnMount(() => {
     const loadSnapshotInfo = async () => {
+      // This is the only .DataRepo.snapshot call. Should it use the catalog?
```

Comment:
Open question! Thread here:

Reply:
Was there supposed to be a link in this comment?

Reply:
No, it is referring to a method in the ajax file. (Would have been more clear to say `Ajax().DataRepo.snapshot.details` call, I can change it)

Reply:
Sorry, "comment" was ambiguous. I was referring to the [GitHub comment](https://github.com/DataBiosphere/terra-ui/pull/2942#discussion_r845479137), not the code comment.

 > Open question! Thread here:

Reply:
Ah, no link intended! The `thread here:` is referring to the comment. I am just looking to open up discussion on it to see what people think.

Reply:
Given that this component is being used in workspaces, I would not change this to use the catalog.